app/models/post.rb

```diff
@@ -0,0 +1,14 @@
+class Post < ApplicationRecord
+  has_many :likes, dependent: :destroy
+  has_many :comments, dependent: :destroy
+  belongs_to :user
+  after_save :update_posts_counter
+
+  def update_posts_counter
```

Comment:
- [ ] OPTIONAL
Following best practices `update_posts_counter` should be marked as _private_ after implementing the `after_save` method.